zetaclient/chains/evm/observer/observer.go

```diff
@@ -40,7 +41,7 @@ import (
 )
 
 // Logger is the logger for evm chains
-// TODO: Merge this logger with the one in bitcoin
+// TODO(revamp): Merge this logger with the one in bitcoin
```

Comment:
same. It's being worked on by creating base structs.

Reply:
I suggest to keep the comment and do a iteration of all of these once we finish the refactorings

Reply:
okay